sdk/formrecognizer/azure-ai-formrecognizer/src/main/java/com/azure/ai/formrecognizer/FormRecognizerAsyncClient.java

```diff
@@ -54,9 +54,10 @@
 import static com.azure.core.util.FluxUtil.monoError;
 
 /**
- * This class provides an asynchronous client that contains all the operations that apply to Azure Form Recognizer.
- * Operations allowed by the client are recognizing receipt, business card, invoice and identity document data from
- * input documents, extracting layout information, analyzing custom forms for predefined data.
+ * This class provides an asynchronous client to connect to the Form Recognizer Azure Cognitive Service to recognize
+ * information from forms and images and extract it into structured data.
+ * <p>It provides methods to analyze receipts, business cards, invoices, to recognize form content, and to
+ * extract fields from custom forms with models trained on custom form types.</p>
  *
  * <p><strong>Instantiating an asynchronous Form Recognizer Client</strong></p>
```

Comment:
The Getting started section and champion scenarios samples can be copied here from Package info.